.test-infra/mock-apis/src/main/java/org/apache/beam/testinfra/mockapis/echo/v1/Echo.java

```diff
@@ -0,0 +1,1447 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.testinfra.mockapis.echo.v1;
```

Comment:
Question: Is all this java, generated code?

Reply:
Yes. However, I had to manually add the suppress warning annotations as the checker.xml configuration wasn't fully working to suppress the Gradle check failures. Kind of a hack that I code commented for others to see. This GitHub repo's proto is oriented around a Java project and a rather opinionated one about proto. In this mock-apis module, the stubbed Go code is more important as, Go is doing more of the heavy lifting here.

Reply:
Ack.

I have no problems with generated code. Just means I won't review it, because it's way out of scope to manually fix things up. If it works it works.